### ISSUE TITLE
Stuck

### ISSUE THREAD

**Issue by IrishReign** [PokemonGoF/PokemonGo-Bot-Desktop#257]
My bot has stopped catching pokemon and now just shows this
![screen shot 2016-09-05 at 1 03 24 pm](https://cloud.githubusercontent.com/assets/21983686/18256360/3fb6c96c-7369-11e6-9a00-108d93f3020e.png)
and claims that its walking forever
El Cap Mac OSX

**Comment by flawnn** [comment]
i think softban. restart and if its there, come again.

@v4c

**Comment by IrishReign** [comment]
@NightScap3 Ok Thanks, you were probably right it stopped oing it after a while

**Comment by IrishReign** [comment]
But normally it catches the softban

**Comment by IrishReign** [comment]
@NightScap3 Yeah it started again, you might be right about the softban, but normally it auto fixes that. I think that if it has been logged out and logged in again it doesn't recognize that its softbanned

**Comment by flawnn** [comment]
its also stucking with fix autoban and i restart it 💯  and it works 😆
